scss/barrier.scss

```diff
@@ -60,15 +62,68 @@ $barrier-padding: 20px;
 	text-align: center;
 }
 
-.amp-access-barrier__offer-price {
-	@include oTypographySans(m);
+.amp-access-barrier__offer-secondary-price {
+	color: oColorsGetPaletteColor('grey-tint3');
+	font-size: 14px;
+	line-height: 16px;
+}
+
+.amp-access-barrier__offer-promo-separator {
+	position: relative;
+    text-align: center;
+	margin: 0 0 8px;
+
+	&:before, &:after {
+		background-color: oColorsGetPaletteColor('black');
+		content: '';
+		display: block;
+		height: 1px;
+		opacity: 0.25;
+		position: absolute;
+		top: 50%;
+		left: 0;
+		width: calc(50% - 20px);
+	}
+
+	&:after {
+		left: unset;
+		right: 0;
+	}
+}
+
+.amp-access-barrier__offer-highlight {
+	background-color: #1aebff; // no similar colour mapped in current version of o-colors
 	color: oColorsGetPaletteColor('black');
+	display: inline-block;
+	margin: 10px auto;
+	padding: 4px 8px;
+	font-size: 14px;
+	line-height: 16px;
+	font-weight: 500;
+}
+
+.amp-access-barrier__offer-price {
+	color: oColorsGetPaletteColor('teal-1');
 	// TODO: Upgrade to latest o-typography
-	line-height: 20px;
-	font-size: 16px;
-	text-align: center;
+	font-size: 20px;
+    line-height: 24px;
```

Comment:
indentation needs adjusting

Reply:
Indentation looks fine as I am looking at the code locally. Maybe we can consider it as an Github issue ?